l10n_nl_partner_salutation/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017-2020 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Full salutation for partners, Dutch style',
+    'version': '12.0.1.0.0',
+    'author': 'Therp BV, Odoo Community Association (OCA)',
+    'category': 'Contact management',
```

Comment:
Please Change category so it shows up corretly in odoo appstore and backend applist.
```"category": "Localization",```
or
```"category": "Localization\Netherlands",```
or
```"category": "Localization\Contact management",```